controllers/backplaneconfig_controller.go

```diff
@@ -1721,9 +1743,7 @@ func (r *MultiClusterEngineReconciler) ensureUnstructuredResource(ctx context.Co
 			return ctrl.Result{}, err
 		}
 		// Creation was successful
-		r.Log.Info(fmt.Sprintf("Created new resource - kind: %s name: %s", u.GetKind(), u.GetName()))
-		// condition := NewHubCondition(operatorsv1.Progressing, metav1.ConditionTrue, NewComponentReason, "Created new resource")
-		// SetHubCondition(&m.Status, *condition)
+		r.Log.Info("Creating resource", "Name", "Kind", u.GetName(), u.GetKind())
```

Comment:
```suggestion
		r.Log.Info("Creating resource", "Name", u.GetName(), "Kind", u.GetKind())
```

Reply:
Updated the code to match this suggestion.